src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  const stateCopy = { ...state };
+
+  for (const a of actions) {
+    switch (a.type) {
```

Comment:
If I were you, I would destructure action properties.

Second thing. I want you to add more descriptive names for variables. `a` -> `action` etc

Reply:
> If I were you, I would destructure action properties.
Could you explain?

Reply:
@mosowiecka sure!

I meant something like this:
```
const { type } = action;
```

Obviously you can destructure more than just `type`